### DISCUSSION TITLE
Querying with count option

### DISCUSSION THREAD

**Discussion by sureshdsk** [supabase/supabase#2764] [Questions] [answered]
I'm checking out if there any way to query postgres using supabase for count, group by based sql.

https://supabase.io/docs/postgrest/client/select#querying-with-count-option

**Comment by steve-chavez** [comment, marked as answer]
To use `group by` right now, you have to create a SQL view or function and call it through  [select](https://supabase.io/docs/reference/javascript/select) or  [rpc](https://supabase.io/docs/reference/javascript/rpc).

**Reply by sureshdsk** [reply]
got it. I've tried used a view to solve this. Thanks
